app/components/arclight/document_component.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class='d-md-flex justify-content-between al-show'>
+  <div class='al-show-breadcrumb'>
+    <nav aria-label="breadcrumb">
+      <%= render breadcrumb_component.new(presenter: presenter) %>
+    </nav>
+  </div>
+</div>
+
+<div class="title-container">
+  <%= content_tag :h1 do %>
+    <%= "Guide to the " + document.normalized_title %>
```

Comment:
Add this to the localisation file for easier re-use and if we ever end up wanting to display other languages.

Please update the text in the `ShowPresenter` when you do.